x/sync/network_server.go

```diff
@@ -35,11 +41,7 @@ const (
 	endProofSizeBufferAmount = 2 * units.KiB
 )
 
-var (
-	_ Handler = (*NetworkServer)(nil)
```

Comment:
was this intended?

Reply:
Yes the `Handler` interface was removed

Reply:
ok yeah missed it
https://github.com/ava-labs/avalanchego/pull/1336/files#diff-19b607d5e23bc408a1e3d9c1727e4844b7038077fe22bedc356b0eb64fcc044dL47